### DISCUSSION TITLE
Support for states holiday

### DISCUSSION THREAD

**Discussion by fikrimastor** [spatie/holidays#181] [Q&A] [answered]
Hi. I'm fikri from Malaysia.

This holiday package only support for country level.

Is there any chance to support for states level?

Because in Malaysia we have several state that only observe several holidays.

Let say I'm working remotely, and currently staying outside my operating office states.

Sometimes if my operating office states on holiday, my staying state does not. And vice versa.

**Comment by Nielsvanpach** [comment, marked as answer]
You can add state holidays too: https://github.com/spatie/holidays?tab=readme-ov-file#adding-a-new-country
